media/css/mozorg/internet-health/privacy-day-egg.scss

```diff
@@ -0,0 +1,231 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../pebbles/includes/lib';
+
+
+#kzp {
+    display: none;
+    bottom: 0;
+    height: 100%;
+    left: 0;
+    overflow: hidden;
+    position: fixed;
+    right: 0;
+    top: 0;
+    width: 100%;
+    z-index: 998;
+}
+
+.kz {
+    background: url('/media/img/mozorg/internet-health/privacy/kz/03-blue.gif') center center no-repeat;
+    @include background-size(cover);
+    height: 119px;
+    position: absolute;
+    top: -500px;
+    width: 248px;
+}
+
+// The Golden Kazoo
+.au {
+    background-image: url('/media/img/mozorg/internet-health/privacy/kz/05-gold.gif');
+    height: 130px;
+    width: 221px;
+
+    a {
+        @include image-replaced;
+        display: block;
+        height: 100%;
+        width: 100%;
+    }
+
+    @media #{$mq-tablet} and #{$mq-tall} {
+        height: 185px;
+        width: 315px;
+    }
+}
+
+// Hide half the kazoos on small screens
+#kz3,
+#kz6,
+#kz8,
+#kz9,
+#kz11,
+#kz14,
+#kz15,
+#kz16 {
+    display: none;
+}
+
+// Big screens get bigger kazoos, in more colors, and more of them.
+// It's a kavalcade of kazoos!
+@media #{$mq-tablet} {
+    .gr {
+        background-image: url('/media/img/mozorg/internet-health/privacy/kz/01-green.gif');
+        height: 275px;
+        width: 155px;
+    }
+
+    .rd {
+        background-image: url('/media/img/mozorg/internet-health/privacy/kz/02-red.gif');
+        height: 300px;
+        width: 240px;
+    }
+
+    .bl {
+        height: 170px;
+        width: 355px;
+    }
+
+    .pr {
+        background-image: url('/media/img/mozorg/internet-health/privacy/kz/04-purple.gif');
+        height: 355px;
+        width: 170px;
+    }
+
+    #kz3,
+    #kz6,
+    #kz8,
+    #kz9,
+    #kz11,
+    #kz14,
+    #kz15,
+    #kz16 {
+        display: block;
+    }
+}
+
+// Trigger the animation on a body class
+body.easter {
+    .kz {
+        animation: kz-fall infinite linear, kz-spin infinite linear;
```

Comment:
We have a mixin for `animation` but none for all the longhand properties like `animation-delay`. I could add those mixins to Pebbles but if we start using PostCSS and autoprefixer they would quickly become redundant. So for this one-off easter egg I think I'll just hardcode all the -webkit prefixes instead of adding a bunch of little-used and short-lived mixins.